src/encoded/schemas/experiment.json

```diff
@@ -268,6 +268,9 @@
         "replicates.library.biosample.life_stage": {
             "title": "Life stage"
         },
+        "replicates.library.biosample.disease_term_name": {
```

Comment:
I have to do this because dev build is broken and there was a merge conflict.  This was the compromise.